server/services/tests/mocks/mock_use_case_helpers.py

```
@@ -0,0 +1,162 @@
+        self._team_avg: Optional[Dict[str, float]] = None
+        self._weakness_vector: Optional[Dict[str, float]] = None
+        self._latest_stats: Optional[Dict[str, float]] = None
+        self._adjustment_sum: float = 0.0
+        self._adjustment_contributions: Dict[str, float] = {}
+    
+    def validate_player_ids(self, player_ids: List[int]) -> None:
+        """validates player ids"""
+        if self._should_raise_validation_error:
+            raise InputValidationError(self._validation_error_message)
+        if not player_ids:
+            raise InputValidationError("Player IDs list cannot be empty")
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)